src/CustomerioTracking.tsx

```diff
@@ -176,6 +176,30 @@ class CustomerIO {
     );
   }
 
+  /**
+   * Track push notifications metrics using this method.
+   * Call this method when a user interacts and taps open the push notification.
+   * @param payload Customer.io payload as received from the push notification
+   */
+  static async trackNotificationResponseReceived(payload: Object) {
+    if (payload == null) {
+      return
+    } 
+    CustomerioReactnative.trackNotificationResponseReceived(payload)
+  }
+
+  /**
+   * Track push notifications metrics using this method.
+   * Call this method when a push notification is received.
+   * @param payload Customer.io payload as received from the push notification
+   */
+  static async trackNotificationReceived(payload: Object) {
+    if (payload == null) {
+      return
+    } 
+    CustomerioReactnative.trackNotificationReceived(payload)
+  }
```

Comment:
Why are these functions `async`? The logic inside of the functions is synchronous so I was not expecting to see these functions be async.

Reply:
Good catch! Fixed it.